### PR TITLE
new version & changelog

### PR DESCRIPTION
**Pull request by rv0lt**
## Read this before submitting the PR

1. Always create a Draft PR first
2. Go through sections 1-5 below, fill them in and check all the boxes
3. Make sure that the branch is updated; if there's an "Update branch" button at the bottom of the PR, rebase or update branch.
4. When all boxes are checked, information is filled in, and the branch is updated: mark as Ready For Review and tag reviewers (top right)
5. Once there is a submitted review, implement the suggestions (if reasonable, otherwise discuss) and request an new review.

If there is a field which you are unsure about, enter the edit mode of this description or go to the [PR template](../.github/pull_request_template.md); There are invisible comments providing descriptions which may be of help.

## 1. Description / Summary

new version and changelog for new release

## 3. Type of change

What _type of change(s)_ does the PR contain?

**Check the relevant boxes below. For an explanation of the different sections, enter edit mode of this PR description template.**

- [ ] New feature
  - [ ] Breaking: _Why / How? Add info here._ 
  - [ ] Non-breaking 
- [ ] Database change: _Remember the to include a new migration version, **or** explain here why it's not needed._ 
- [ ] Bug fix 
- [ ] Security Alert fix 
  - [ ] Package update 
    - [ ] Major version update 
- [x] Documentation 
- [ ] Workflow 
- [ ] Tests **only** 

## 4. Additional information

- [x] [Sprintlog](../SPRINTLOG.md) 
- [ ] Blocking PRs 
  - [ ] Merged 
- [ ] PR to `master` branch: \_If checked, read [the release instructions](../doc/procedures/new_release.md) 
  - [ ] I have followed steps 1-8. 

## 5. Actions / Scans

_Check the boxes when the specified checks have passed._

**For information on what the different checks do and how to fix it if they're failing, enter edit mode of this description or go to the [PR template](../.github/pull_request_template.md).**

- [x] **Black**

- [x] **Prettier**

- [x] **Yamllint**

- [x] **Tests**

- [x] **CodeQL**

- [x] **Trivy**

- [x] **Snyk**

### CODE REVIEW

File: CHANGELOG.rst
Comment (i-oden) [inline CHANGELOG.rst:15]:
```suggestion
- Logging:
```

File: CHANGELOG.rst
Comment (i-oden) [inline CHANGELOG.rst:17]:
```suggestion
    - Username logged when password is reset.
```

**Review by i-oden — CHANGES_REQUESTED** [review]
Change order so that it's consistent with the rest of the changelog, and also so that the users get the most important information first:

- New features
- Bugs fixed
- Logging

The other comments I've made are less important, mainly for simplification purposes

File: CHANGELOG.rst
Comment (valyo) [inline CHANGELOG.rst:12]: Is this a new feature, maybe it fits better in bug fixes

**Review by valyo — CHANGES_REQUESTED** [review]

File: SPRINTLOG.md
Comment (i-oden) [inline SPRINTLOG.md:470]:
```suggestion
- New version: 2.9.0 ([#1583](https://github.com/ScilifelabDataCentre/dds_web/pull/1583))
```

**Review by i-oden — CHANGES_REQUESTED** [review]
I merged your PR regarding the CVEs so update the changelog regarding that as well.

File: CHANGELOG.rst
Comment (i-oden) [inline CHANGELOG.rst:6]:
```suggestion
2.9.0 - 2024-12-18
```

File: CHANGELOG.rst
Comment (i-oden) [inline CHANGELOG.rst:18]: Think of this section from the users perspective: If you were the user, how would you best understand the changes. For me, this would be the following. 
```suggestion
- New features:
    - Documentation updated regarding 'Upload' or 'Download' at the end of staging directory 
    - The unit's contact email is now also notified when the unit is approaching the allocated data quota
- Bugs fixed:
    - Change in error if there is a download attempt after the password has been reset
    - The 'dds ls --tree' command works again
    - The mariadb-client version is now pinned in the dockerfile
- Logging:
    - The exception raised when there's an invalid token has been removed to simplify logging
    - Username logged when password is reset
    - MaintenanceModeException is filtered out from the logs
```

Comment (i-oden) [reply CHANGELOG.rst:18]: Also, don't forget to add a row regarding the CVE's.

**Review by i-oden — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst
Comment (i-oden) [inline CHANGELOG.rst:6]: The new version is being deployed on the 18th.

Comment (i-oden) [reply CHANGELOG.rst:6]:
```suggestion
2.9.0 - 2024-12-18
```

File: SPRINTLOG.md
Comment (i-oden) [inline SPRINTLOG.md:471]: This needs to be moved to the correct sprint

**Review by i-oden — CHANGES_REQUESTED** [review]
Change the date in the changelog and move the row in the sprintlog to the correct sprint

File: SPRINTLOG.md
Comment (i-oden) [reply SPRINTLOG.md:471]:
```suggestion

# 2024-12-16 - 2024-12-20

- New version: 2.9.0 ([#1583](https://github.com/ScilifelabDataCentre/dds_web/pull/1583))
```

**Review by i-oden — APPROVED** [review]
You're good to merge once actions have passed and @valyo has approved

**Comment by i-oden** [comment]
actions fixed in: https://github.com/ScilifelabDataCentre/dds_web/pull/1584

**Pull request closed without merging**
